lib/rides/pages/search_suggestion_page.dart

```diff
@@ -64,8 +64,11 @@ class _SearchSuggestionPage extends State<SearchSuggestionPage> {
       lastDate: firstDate.add(const Duration(days: 30)),
     ).then((value) {
       if (value != null) {
-        _selectedDate = DateTime(value.year, value.month, value.day, _selectedDate.hour, _selectedDate.minute);
-        _dateController.text = localeManager.formatDate(_selectedDate);
+        setState(() {
+          _selectedDate = DateTime(value.year, value.month, value.day, _selectedDate.hour, _selectedDate.minute);
+          _dateController.text = localeManager.formatDate(_selectedDate);
+        });
+        if (_timeValidator(_timeController.text) == null) loadRides();
```

Comment:
why does only the time need to be validated? And when the time is wrong, shouldn't we show some kind of error message?

Reply:
because the date is validated by the picker which only allows to select dates in the future. And there is a error message displayed. Thats what the errorText is for